en/faq/postcss-plugins.md

```diff
@@ -5,7 +5,8 @@ description: How to add PostCSS plugins?
 
 # How to add PostCSS plugins?
 
-In your `nuxt.config.js` file:
+### Default Method
+Delete the `postcss.config.js` in your project directory (if existent). Then, in your `nuxt.config.js` file add the following:
```

Comment:
```suggestion
If present, rename or delete the `postcss.config.js` in your project directory. Then, in your `nuxt.config.js` file add the following:
```

Reply:
@farzadso what do you think about it?

Reply:
@core01 Couldn't agree any more. It's got much better sentencing.